pipelines/README.md

```diff
@@ -1,88 +1,90 @@
-the source. The data warehouse can be a collection of
+or. The data warehouse can be a collection of
 [Apache Parquet files](https://parquet.apache.org), or it can be another FHIR
 server (e.g., a HAPI FHIR server or Google Cloud FHIR store) or eventually a
-cloud based datawarehouse (like [BigQuery](https://cloud.google.com/bigquery)).
+cloud based data warehouse (like [BigQuery](https://cloud.google.com/bigquery)).
 
 There are four modes of transfer:
 
 - [**Batch mode (FHIR)**](#batch-mode-using-fhir-search): This mode uses FHIR
   Search APIs to select resources to copy, retrieves them as FHIR resources, and
   transfers the data via FHIR APIs or Parquet files.
-- [**Batch mode (JDBC)**](#batch-mode-using-jdbc): For an OpenMRS source, this
-  mode uses JDBC to read the IDs of entities in an OpenMRS MySQL database,
-  retrieves those entities as FHIR resources, and transfers the data via FHIR
-  APIs or Parquet files. For a HAPI source, this mode uses JDBC to read FHIR
-  resources directly from a HAPI database, and transfers the data.
+- [**Batch mode (JDBC)**](#batch-mode-using-jdbc): For a HAPI source, this mode
+  uses JDBC to read FHIR resources directly from a HAPI database, and transfers
+  the data. For an OpenMRS source, this mode uses JDBC to read the IDs of
+  entities in an OpenMRS MySQL database, retrieves those entities as FHIR
+  resources, and transfers the data via FHIR APIs or Parquet files.
 - **[Streaming mode (Debezium)](#streaming-mode-debezium)**: This mode
   continuously listens for changes to the underlying OpenMRS MySQL database
   using
   [Debezium](https://debezium.io/documentation/reference/1.2/connectors/mysql.html).
 
 There is also a query module built on top of the generated data warehouse.
 
-## Prerequisites
-
-- An [OpenMRS](https://openmrs.org) instance with the latest version of the
-  [FHIR2 Module](https://addons.openmrs.org/show/org.openmrs.module.openmrs-fhir2-module)
-  installed, or a [HAPI](https://hapifhir.io/) instance.
-  - There is an [OpenMRS Reference Application](#run-openmrs-using-docker) image
-    with these prerequisites and demo data you can use to try things out.
-  - There is a [HAPI FHIR source server and database](/docker/hapi-compose.yml)
-    image you can use for testing.
-- A target output for the data. Supported options are Apache Parquet files or a
-  FHIR server such as HAPI FHIR or
-  [Google Cloud Platform FHIR stores](https://cloud.google.com/healthcare/docs/how-tos/fhir).
-  - You can use our [HAPI FHIR server](#run-hapi-fhir-server-using-docker) image
-    for testing FHIR API targets.
-  - [Learn how to create a compatible GCP FHIR store](#create-a-google-cloud-fhir-store-and-bigquery-dataset),
-    if you want to use this option.
-- Build the [local version of Bunsen](bunsen/)
-
-### Installing the FHIR2 Module
-
-**To install via web**: Log in to OpenMRS as an administrator, go to **System
```

Comment:
Oh my ... these documentations were so old! @williamito please make sure that if any Wiki pages were created based on these old documentations, they are updated with @omarismail94 changes (thank you).

Reply:
@williamito  I only resolved this so I can merge the PR